### PR TITLE
@eessex => Publishing canvas ad height

### PR DESCRIPTION
**Pull request by damassi**
Addresses https://github.com/artsy/publishing/issues/185

### CODE REVIEW

File: src/Components/Publishing/Display/Canvas/CanvasContainer.tsx [hunk 1]
```diff
@@ -89,7 +89,7 @@ export class CanvasContainerComponent extends React.Component<CanvasContainerPro
       const slideshowProps = { unit, campaign, disclaimer, containerWidth: size.width }
 
       return (
-        <CanvasSlideshow {...slideshowProps}>
+        <CanvasSlideshow {...slideshowProps} renderTitleCard>
```
Comment (damassi) [inline src/Components/Publishing/Display/Canvas/CanvasContainer.tsx]: This opens up the space for the mobile slideshow stuff to show but it's still hidden for reasons I can't quite figure out.

File: src/Components/Publishing/__stories__/Article.story.tsx [hunk 2]
```diff
@@ -70,6 +71,52 @@ ads.forEach(mediaType => {
...
+        display={Display('image')}
+        relatedArticlesForPanel={RelatedPanel}
+        relatedArticlesForCanvas={RelatedCanvas}
+        emailSignupUrl="#"
+      />
+    </div>
+  )
+})
+
+const Break = styled.div`
```
Comment (damassi) [inline src/Components/Publishing/__stories__/Article.story.tsx]: Grabbed this from force code so we would have a better testbed for seeing how infinite scroll is displayed

File: src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx [hunk 3]
```diff
@@ -15,6 +15,7 @@ interface CanvasSlideshowProps {
   containerWidth: number
   disclaimer: any
   unit: any
+  renderTitleCard?: boolean
```
Comment (eessex) [inline src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx]: it is not clear to me where the renderTitleCard prop is being used

Comment (damassi) [reply src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx]: It's being used here: https://github.com/damassi/reaction/blob/master/src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx#L151

Comment (eessex) [reply src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx]: Does it not get redefined here? https://github.com/damassi/reaction/blob/master/src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx#L143

Comment (eessex) [reply src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx]: but i think it is redefined just above https://github.com/damassi/reaction/blob/master/src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx#L144

Comment (damassi) [reply src/Components/Publishing/Display/Canvas/CanvasSlideshow.tsx]: Ah, you are correct -- will fix.

**Comment by kanaabe** [comment]
Nice!

**Pull request merged**